src/scicookie/{{cookiecutter.project_slug}}/conda/dev.yaml

```
@@ -7,3 +7,4 @@ dependencies:
   - poetry
   - nodejs  # used by semantic-release
   - shellcheck
+  - pandoc
```

@xmnlab @YurelyCamacho I think it should be:

```suggestion
{% if cookiecutter.documentation_engine == 'sphinx' -%}
  - pandoc
 {% endif %}
```
Am I right?

---

@YurelyCamacho could implement this idea @Anavelyz suggested?
please just to ensure to use the jinja tags correctly .. maybe:

```yaml
{% if cookiecutter.documentation_engine == 'sphinx' -%}
	  - pandoc
{%- endif %}
```

---

@YurelyCamacho maybe you can try this:

```yaml
{%- if cookiecutter.documentation_engine == 'sphinx' %}
  - pandoc
{%- endif %}
```

btw, I recommend you to play with: http://jinja.quantprogramming.com/

for this case, you can use as `value (yaml)`, this input:

```yaml
cookiecutter:
  project_slug: test
  documentation_engine: sphinx
```